app/Models/Topic.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletes;
+
+class Topic extends Model
+{
+    use SoftDeletes;
+    /**
+     * [$table description]
+     * @var string
+     */
+    protected $table= 'topics';
+    /**
+     * [$dates description]
+     * @var array
+     */
+    protected $dates = ['deleted_at'];
+    /**
+     * [$fillable description]
+     * @var [type]
+     */
+    protected $fillable = [
+        'name',
+        'slug',
+        'images',
+        'set_time',
+        'select_time',
+        'status',
+        'created_at',
+        'updated_at',
+        'deleted_at'
+    ];
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoCommaAfterLast</br>**Severity**: error</br>**File**: app/Models/Topic.php</br>Comma required after last value in array declaration